react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export default class CustomButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: 0 };
```

Comment:
Not a necessary fix, but in general variables / properties named `isSomething` sound like they should contain a boolean value.